modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -1,23 +1,19 @@
 // Module included in the following assemblies:
 //
-// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+// * scalability_and_performance/ztp-far-edge/ztp-reference-cluster-configuration-for-vdu.adoc
 
-:_content-type: PROCEDURE
+:_content-type: CONCEPT
 [id="sno-du-enabling-workload-partitioning_{context}"]
-= Enabling workload partitioning
+= Workload partitioning
```

Comment:
You could expand this heading to be more descriptive. This will enhance findability.